NEWS.md

```diff
@@ -1 +1,15 @@
 # SVS 0.0.7 Release Notes
+
+## Additions and Changes
+
+* Implemented batch iterator support for hybrid search
+
+* Added support for custom threading and memory allocation
+
+* Introduced a timeout feature for search calls
+
+* Introduced `reuse_empty` flag in dynamic Vamana, enabling users to choose whether to reuse empty entries that may exist after deletion and consolidation
+
+* Enhanced heuristics in the Vamana construct to improve efficiency when adding a small number of points.
+
+* Added several threadpool implementation for users to choose
```

Comment:
maybe we can remove the last point as it's covered by custom threading (2nd point)